src/Microsoft.Win32.SystemEvents/src/Microsoft/Win32/SystemEvents.cs

```diff
@@ -560,13 +441,11 @@ private void Dispose()
                     }
                 }
 
-                // If DestroyWindow failed, it is because we're being
-                // shutdown from another thread.  In this case, locate the
-                // DefWindowProc call in User32, set the window proc to call it,
-                // and post a WM_CLOSE.  This will close the window from 
-                // the correct thread without relying on managed code executing.
                 if (Interop.User32.IsWindow(handle) && !Interop.User32.DestroyWindow(handle))
                 {
+                    // We may not have been able to destroy the window if we're shutdown from another thread.
+                    // Attempt to close the window by sending the WM_CLOSE message instead. (Messages always
```

Comment:
Given that sending and posting do have different behaviour might it be a good idea to change the comment to agree with the method name called?

Reply:
Sure